pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -1100,6 +1100,7 @@ type KubevirtCloudSpec struct {
 	CSIKubeconfig string `json:"csiKubeconfig,omitempty"`
 
 	PreAllocatedDataVolumes []PreAllocatedDataVolume `json:"preAllocatedDataVolumes,omitempty"`
+	InfraStorageClasses     []string                 `json:"infraStorageClasses,omitempty"`
```

Comment:
Please add a comment for this field and Pre-allocated data volume field. Eventually we will add comments to each field.